test/integration/patients/worklist/worklist.js

```diff
@@ -392,8 +392,8 @@ context('worklist page', function() {
           name: 'First In List',
           details: 'Like the legend of the phoenix All ends with beginnings What keeps the planet spinning The force from the beginning Look We\'ve come too far To give up who we are So let\'s raise the bar And our cups to the stars',
           duration: 0,
-          due_date: null,
-          due_time: null,
+          due_date: testDateAdd(3),
```

Comment:
If we just change this to `testDate()`, the test passes. I don't know why I did `testDateAdd(3)` in the first place...? It doesn't seem necessary since we're just testing the `due_time`.